sympy/stats/stochastic_process_types.py

```diff
@@ -296,26 +326,27 @@ def _extract_information(self, given_condition):
         """
         if isinstance(self, DiscreteMarkovChain):
             trans_probs = self.transition_probabilities
+            state_index = self.state_index
```

Comment:
What's the motivation behind, using `state_index` and not `state_space`? Actually, `state_space` is public, so if possible please restore the previous name.